code/datums/tips/tips.dm

```diff
@@ -0,0 +1,341 @@
+		"Toolboxes do more damage if they are full of things, but their contents will spill when you use them as a weapon.",
+		"Heads of Staff will be more willing to help you if you bring paperwork that's already filled out.",
+		"Chainswords and energy weapons can slice both walls and people.",
+		"You can use . (period) instead of : to speak into a radio.",
+		"The radio key .i will allow you to speak into a nearby intercom, .r will speak into a radio in your right hand, and .l will speak into your left. The microphone does not need to be enabled for this to work.",
+		"Your ID card's access determines what departmental channels you can set intercoms to.",
+		"You can fax papers between requests consoles by attacking the console with a paper. Make sure the paper tray is closed first!",
+		"No wall or window is 100% impervious to heat.",
+		"Never try to fight the hostile maintenance drones on your own.",
+		"There is no cloning, but you can respawn... as a different person, obviously.",
+		"The Newscasters are actively updated with Aurora lore... please read them.",
+		"Your PDA messages can't be erased, so use them wisely.",
+		"The rules are there to help, read them today!",
+		"Role-playing isn't easy, the wiki contains resources to help.",
+		"Trash piles are bounties of interesting loot, but you can also hide in them, just drag your sprite onto the pile."
+	)
+
+/*
+	MEME TIPS
+*/
+
+/datum/tip/meme
+	category = CATEGORY_MEME
+	messages = list(
+		"Occasionally the tip of the round might lie to you. Don't panic, this is normal.",
+		"To defeat your enemy, click on them until they go horizontal.",
+		"Killing the entire ship isn't fun except when it is.",
+		"You can win a pulse rifle from the arcade machine. Honest.",
+		"Just like real life the entropy of the game can only increase with time. If things aren't on fire yet, just wait.",
+		"Cleanbot.",
+		"It's fun to try and predict the round type from the tip of the round message.",
+		"The Quartermaster has finally become a Head of Staff. He only needed to be renamed to Operations Manager. It can be that easy, kids.",
+		"Your sprite represents your hitbox, so that floor length braid makes you easier to kill. The sacrifices we make for snowflake points.",
+		"Spacemen can't move diagonally but most animals can. Blame the slow decline of the numpad.",
+		"The remake might come out.",
+		"DEATH IS IMMINENT!",
+		"Just the tip?",
+		"Phoron check.",
+		"Remember the cheesy line!",
+		"See no evil, hear no evil, speak no evil.",
+		"Service borgs can use the special maidborg sprite if they pray hard enough.",
+		"If you see this message something has gone seriously wrong.",
+		"If you're having trouble as an antagonist, consider a Dionaea whitelist.",
+		"The second antag contest is still running, you know.",
+		"Killing rats on sight is not and never will be gank, no matter how much people complain.",
```

Comment:
```suggestion
		"Killing rats on sight is not and never will be gank, no matter how much people complain. Not that this matters, since you can't even spawn as a rat.",
```